airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -120,7 +114,7 @@ def get_task_instances(
     end_date_lte: Optional[str] = None,
     duration_gte: Optional[float] = None,
     duration_lte: Optional[float] = None,
-    state: Optional[str] = None,
+    state: Optional[List[str]] = None,
```

Comment:
So the previous annotation was wrong, right? The value passed would would be `List[str]` even before this change, this simply fixes the type annotation to match reality, from what I can tell.

Reply:
Correct, it shouldn't change anything (apart from making some static linters feel better)